src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -0,0 +1,17 @@
+package wolox.training;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@EnableJpaRepositories("wolox.training.repositories")
```

Comment:
This annotation is also unnecessary for the same reason as the entityscan one :)